test/integration/test_integration.py

```diff
@@ -2038,8 +2038,8 @@ def forward(self, x):
         self.assertTrue(torch.equal(after_export, ref))
         if api is _int8da_int4w_api:
             targets = [n.target for n in model.graph.nodes]
-            self.assertTrue(torch.ops.quant.choose_qparams_affine.default in targets)
-            self.assertTrue(torch.ops.quant.quantize_affine.default in targets)
+            self.assertTrue(torch.ops.torchao.choose_qparams_affine.default in targets)
+            self.assertTrue(torch.ops.torchao.quantize_affine.default in targets)
```

Comment:
Are these numerically equivalent to the `torch.ops.quant` counterparts? Should we add tests to show that?

Reply:
yeah the ones in pytorch are copy pasted from torchao so should be the same